lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -46,6 +55,7 @@ def analyze(changes)
     def generate_issue(path)
       loc = lines_of_code[path]
       commit = last_committed_at.fetch(path)
+      churn = code_churn[path.to_s]
```

Comment:
[suggestion] How about using `Pathname` for consistency with `lines_of_code`?

```suggestion
      churn = code_churn[path]
```

Reply:
Fixed. commit 6401b0e79efb305c9d9183614e9352754fbf6660